toolkits/global/packages/global-autocomplete/js/index.js

```diff
@@ -78,40 +78,33 @@ const autoComplete = arguments_ => {
 			let previousSibling = activeElement.previousSibling;
 			let currentIndex = Array.from(activeElement.parentNode.children).indexOf(activeElement);
 
-			switch (event.key) {
-				case 'ArrowDown':
-					if (nextSibling) {
-						if (selectOnSuggestionBrowsing) {
-							if ((currentIndex + 1) < suggestions().length) {
-								input.value = nextSibling.textContent;
-							} else {
-								input.value = currentSearchTerm;
-							}
-						}
-						nextSibling.focus();
-					}
-					break;
-
-				case 'ArrowUp':
-					if (previousSibling) {
-						if (selectOnSuggestionBrowsing) {
-							input.value = previousSibling.textContent;
-						}
-						previousSibling.focus();
-					} else {
-						input.focus();
-						if (selectOnSuggestionBrowsing) {
+			if (/ArrowDown|Down/.test(event.key)) {
+				if (nextSibling) {
+					if (selectOnSuggestionBrowsing) {
+						if ((currentIndex + 1) < suggestions().length) {
+							input.value = nextSibling.textContent;
+						} else {
 							input.value = currentSearchTerm;
 						}
 					}
-					break;
-
-				case 'Escape':
-					removeSuggestions();
-					input.value = currentSearchTerm;
-					break;
-				default:
-					break;
+					nextSibling.focus();
+				}
+			} else if (/ArrowUp|Up/.test(event.key)) {
+				if (previousSibling) {
+					if (selectOnSuggestionBrowsing) {
+						input.value = previousSibling.textContent;
+					}
+					previousSibling.focus();
+				} else {
+					input.focus();
+					if (selectOnSuggestionBrowsing) {
+						input.value = currentSearchTerm;
+					}
+				}
+			}
+			else if (event.key === 'Escape') {
```

Comment:
```suggestion
			} else if (event.key === 'Escape') {
```

Another lint issue.

Almost complained about those one, but I thought this passed through the linter already, and that you were leveraging the syntax for readability sake. Sorry.

Reply:
yeah I should have run the lint script before committing - always forget!
I'm used to having lint and test run together..